pkg/controllers/cloud/networkpolicy_sync.go

```diff
@@ -255,6 +255,7 @@ func (r *NetworkPolicyReconciler) syncWithCloud() {
 	log := r.Log.WithName("CloudSync")
 
 	if r.bookmarkCnt < npSyncReadyBookMarkCnt {
+		log.V(1).Info("Skipping sync wait for bookmark events to be processed")
```

Comment:
Dont add this, otherwise on restart for each ANP, it will be logged

Reply:
Done